postcss.config.js

```diff
@@ -1,39 +1,21 @@
 const path = require('path');
 const postCSSImport = require('postcss-import');
-const postCSSPresetEnv = require('postcss-preset-env');
 const postCSSSvg = require('postcss-svg');
 const svgoPlugins = require('./webpack.base').svgoPlugins;
 const tailwindcss = require('tailwindcss');
+const nesting = require('tailwindcss/nesting');
 const autoprefixer = require('autoprefixer');
 
-// Aliases for CSS @import rules (relative to this file)
-const aliases = {
-  'variables.css': './src/styles/variables.css',
-};
-
 module.exports = {
   plugins: [
-    postCSSImport({
-      resolve(id) {
-        if (typeof aliases[id] == 'string') {
-          return path.resolve(__dirname, aliases[id]);
-        }
-        return id;
-      },
-    }),
-    postCSSPresetEnv({
-      features: {
-        'nesting-rules': true,
```

Comment:
Nesting i now done by the tailwind nesting plugin and the color-mod function was never used, so we could remove the entire thing